src/write/endian_vec.rs

```diff
@@ -0,0 +1,107 @@
+use vec::Vec;
+
+use endianity::Endianity;
+use write::{Error, Result, Writer};
+
+/// A `Vec<u8>` with endianity metadata.
```

Comment:
Worth pointing out in the doc comment that the raison d'etre for `EndianVec` is that it is a `Writer`.